packages/@expo/cli/e2e/__tests__/export.test.ts

```diff
@@ -70,6 +70,12 @@ describe('server', () => {
 
   beforeAll(async () => {
     projectRoot = await setupTestProjectWithOptionsAsync('basic-export', 'with-assets');
+
+    // TODO(cedric): Remove this once we publish `@expo/metro-config` with `export --dev` fixes
+    // Or when we can build `@expo/metro-config` on Windows
+    const srcMetroConfig = path.resolve(__dirname, '../../../metro-config/build');
+    const destMetroConfig = path.join(projectRoot, 'node_modules/@expo/metro-config/build');
+    await fs.cp(srcMetroConfig, destMetroConfig, { recursive: true, force: true });
```

Comment:
This is a workaround originally created by Kudo. I know we do have `linkExpoPackages` in the `setupTestProjectWithOptionsAsync` that generates a tarball from source, and adds it as dependency.

Unfortunately, `@expo/metro-config` uses `expo-module-scripts`, which doesn't work on Windows. When adding this library to that list, _and_ add a resolution to avoid different transitive dependencies, Windows tests will fail because it can't actually generate the tarball on Windows (because it can't build it).

This is a very annoying limitation that has affected a lot already, and is now affecting what we can test in CLI's E2E tests. I think it's time to revise this and make `expo-module-scripts` work on Windows.